src/config/laravel-analytics.php

```diff
@@ -20,8 +20,21 @@
     'cache_lifetime_in_minutes' => 60 * 24,
 
     /*
-     * The directory where the underlying Google_Client will store it's cache files.
-     */
-    'cache_location' => storage_path('app/laravel-google-analytics/google-cache/'),
+    |--------------------------------------------------------------------------
+    | Google_client's cache configuration
+    |--------------------------------------------------------------------------
+    |
+    | Here you may configure the "store" that the underlying Google_Client will
+    | use to store it's data.
+    | You may also add extra parameters that will be passed on setCacheConfig (see docs for google-api-php-client).
+    |
+    | Optional parameters: "lifetime", "prefix"
```

Comment:
Correct me if I'm wrong but I don't see any code that processes the `lifetime` and `prefix` parameters.